drivers/scheduler/k8s/k8s.go

```diff
@@ -81,6 +81,8 @@ const (
 
 var (
 	namespaceRegex = regexp.MustCompile("{{NAMESPACE}}")
+	// SecretConfigMap is name of the config map that will store px secrets with auth tokens
+	SecretConfigMap string
```

Comment:
Doesn't need to be exposed from the package and move it to the K8s struct below.